build/debs/build.go

```diff
@@ -16,20 +16,24 @@ import (
 	"github.com/blang/semver"
 )
 
-type ChannelType string
+type DistributionType string
 
 const (
-	ChannelStable   ChannelType = "stable"
-	ChannelUnstable ChannelType = "unstable"
-	ChannelNightly  ChannelType = "nightly"
+	DistributionStable   DistributionType = "stable"
+	DistributionUnstable DistributionType = "unstable"
+	DistributionTesting  DistributionType = "testing"
```

Comment:
Aaah in a later patch you reorder to unstable, testing, stable.  So unstable is some type of periodic (nightly?), testing is something that's promoted out of the periodic for broader test consumption, and stable is something that's promoted out of test?

Reply:
@tpepper -- yep, nailed it! I based the naming on the [Debian package streams](https://www.debian.org/distrib/packages):
- `stable`: "This is stable and well tested software, which changes only if major security or usability fixes are incorporated."
- `testing`: "This area contains packages that are intended to become part of the next stable distribution. There are strict criteria a package in unstable (see below) must obey before it can be added to testing. Note that testing does not get the timely security updates from the security team."
- `unstable`: "This area contains the most recent packages in Debian. Once a package has met our criterion for stability and quality of packaging, it will be included in testing. unstable is also not supported by the security team. Packages in unstable are the least tested and may contain problems severe enough to affect the stability of your system. Only experienced users should consider using this distribution."

Reply:
You may notice I'm not the huge-est fan of the Debian influenced naming.  I just don't feel "unstable" is accurate, when it's more typically a periodic build.  If we're doing things right, the periodics are stable, do then promote to testing, and do then to promote to stable.

Reply:
While these packages are for Debian, our channels should be named consistently for our project and its users.  For those who are Debian users the Debian names might be familiar, but they may not follow the principle of least surprise for other OS users or be appropriate for our intended build/publish/promote workflow.